src/Compilers/Extension/source.extension.vsixmanifest

```diff
@@ -6,7 +6,7 @@
     <Description xml:space="preserve">Package of the Roslyn compilers that enables them for Visual Studio builds.</Description>
   </Metadata>
   <Installation Experimental="true">
-    <InstallationTarget Version="[14.0,]" Id="Microsoft.VisualStudio.Pro" />
+    <InstallationTarget Version="[15.0,]" Id="Microsoft.VisualStudio.Pro" />
```

Comment:
This is an open ended range hence didn't we already say we supported Dev16?

Reply:
Keep reading... :smile: